docs/architecture-decisions/adr010-luxon-date-library.md

```diff
@@ -0,0 +1,46 @@
+---
+id: adrs-adr010
+title: ADR010: Use the Luxon Date Library
+description: Architecture Decision Record (ADR) for Luxon Date Library
+---
+
+| Created    | Status |
+| ---------- | ------ |
+| 2021-01-16 | Open   |
+
+# ADR010: Luxon Date Library
+
+## Status
+
+Proposed
```

Comment:
I'm guessing this can be updated to `accepted`?

Reply:
We discussed this briefly and landed on removing status entirely for ADRs since we haven't had the need for status in the last ADRs. The PR is pretty much the propose step. Bigger things should possibly be structured as an RFC

Reply:
@jhaals - I see @Rugvip removed the status frontmatter in #4140, but the `Status` header still exists in the template. I'm guessing that should also be removed?

Reply:
Yeah good point, we should make sure the template is updated too 👍

Reply:
Oh didn't notice the duplication there in the template. Yeah let's get rid of that as well